gpflow/training/scipy_optimizer.py

```diff
@@ -54,15 +53,18 @@ def make_optimize_tensor(self, model, session=None, var_list=None, **kwargs):
             model.initialize(session=session)
             return optimizer
 
-    def minimize(self, model, session=None, var_list=None, feed_dict=None,
-                 maxiter=1000, disp=False, initialize=False, anchor=True, **kwargs):
+    def minimize(self, model, session=None, var_list=None, feed_dict=None, maxiter=1000,
+                 disp=False, initialize=False, anchor=True, step_callback=None, **kwargs):
```

Comment:
I know, it will sound strange, but could you add type annotation for step_callback?

Reply:
@ahsimb highlight the callback signature